### PR TITLE
Tabs support

### PR DESCRIPTION
**Pull request by dru-karimcossutti**

### CODE REVIEW

File: src/lupus-slider-tabs.vue [hunk 1]
```diff
@@ -0,0 +1,100 @@
+<template>
```
Comment (maximilianmikus) [inline src/lupus-slider-tabs.vue:1]: I think you have 4 spaces configured for indentation

**Review by maximilianmikus — CHANGES_REQUESTED** [review]

File: src/lupus-slider.vue [hunk 2]
```diff
@@ -102,6 +155,12 @@
       this.transformSlide(this.$refs.lupusSlider.swiper.activeIndex + 1);
     },
     methods: {
+      slideTo(slide, speed, callback) {
```
Comment (maximilianmikus) [inline src/lupus-slider.vue]: `callback` is not used

File: src/lupus-slider-tabs.vue (hunk 1 above)
Comment (dru-karimcossutti) [reply src/lupus-slider-tabs.vue:1]: Corrected indentation

File: src/lupus-slider.vue (hunk 2 above)
Comment (dru-karimcossutti) [reply src/lupus-slider.vue]: I know but it's nice to have as a feature, being able to perform some action after slide change

Comment (maximilianmikus) [reply src/lupus-slider.vue]: I agree, but it won't be called, I don't see it used anywhere?

**Review by maximilianmikus — APPROVED** [review]

**Pull request merged**